src/Provider/Doctrine/FixtureFactory.php

```diff
@@ -105,13 +111,22 @@ public function get($name, array $fieldOverrides = [])
             $config['afterCreate']($ent, $fieldValues);
         }
 
-        if ($this->persist) {
+        if ($this->persist && self::canPersist($entityMetadata)) {
             $this->em->persist($ent);
         }
 
         return $ent;
     }
 
+    private static function canPersist(Mapping\ClassMetadata $classMetadata): bool
+    {
+        if (!property_exists($classMetadata, 'isEmbeddedClass')) {
```

Comment:
This library requires doctrine/orm 2.6.3. The property should be available in all orm versions since then: https://github.com/doctrine/orm/blob/v2.6.3/lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php#L270